specgen/specgen/hellocap/hellocap.go

```diff
@@ -0,0 +1,57 @@
+// Package hellocap provides facilities for capturing TLS ClientHellos.
+package hellocap
```

Comment:
This package is a little funky.  But the very idea here is funky: we offer an HTTPS server which can reach down through the protocol stack to capture bytes at the TCP level.